server/src/main/java/org/opensearch/cluster/routing/RoutingNodes.java

```diff
@@ -97,6 +97,7 @@
 
     private final Map<String, ObjectIntHashMap<String>> nodesPerAttributeNames = new HashMap<>();
     private final Map<String, Recoveries> recoveriesPerNode = new HashMap<>();
+    private final Map<String, Recoveries> initialRecoveriesPerNode = new HashMap<>();
     private final Map<String, Recoveries> primaryRecoveriesPerNode = new HashMap<>();
```

Comment:
Can we rename this better `primaryRecoveriesPerNode ` as `initialPrimaryRecoveries` and `initialRecoveriesPerNode ` as `initialReplicaRecoveries`

Reply:
Will rename.